src/components/Questionnaire/ValueSetSelect.tsx

```diff
@@ -81,7 +212,19 @@ export default function ValueSetSelect({
         onValueChange={setSearch}
         autoFocus
       />
-      <CommandList className="overflow-y-auto">
+      <CommandList className="h-[300px] overflow-hidden">
```

Comment:
let's not use arbitrary pixel values. use rem or standard values instead; this does not go well on different DPI settings